pfcwd/main.py

```diff
@@ -339,6 +337,10 @@ def stop(self, ports):
         )
 
         if len(ports) == 0:
+            click.echo("Ports not selected")
+            return
+
+        if 'all' in ports:
```

Comment:
should use 'all' == ports?

Reply:
Such logic with the "in" statement should handle the case: "config pfcwd stop Ethernet0 Ethernet1 all" and remove all ports despite the first two arguments

Reply:
I don't think we need to support this kind of command: "config pfcwd stop Ethernet0 Ethernet1 all"  seems strange to me.